src/app/home-page-top-section/TopSection.tsx

```diff
@@ -0,0 +1,35 @@
+import { Container, Title, Text, Image } from "@mantine/core";
+
+export default function TopSection() {
+  {
+    console.log(Text.classes);
+  }
+  const container_style = {
+    padding: "0",
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "center",
+    alignItems: "center",
+    margin: "50px auto 5% auto",
+    color: "#0078e8",
+    textAlign: "center",
+    paddingBottom: "20px"
+  } as const;
```

Comment:
is const as a variable used anywhere? If not we can remove this
Plus the naming is like `const` the type